city_scrapers/spiders/chi_ssa_35.py

```diff
@@ -0,0 +1,122 @@
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Commission"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        item_url = item[1].css("::attr(href)").get()
+        response = requests.get(item_url)
+
+        lp = LAParams(line_margin=0.1)
+        output_str = StringIO()
+        extract_text_to_fp(BytesIO(response.content), output_str, laparams=lp)
+        pdf_text = output_str.getvalue().replace("\n", "")
+
+        return pdf_text
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return COMMISSION
+
+    def _clean_date_item(self, date_item):
+        date_item = date_item.split()
+
+        if len(date_item) == 5:
+            date_item[-1] = date_item[-1].replace(")", "")
+            date_item[-2] = date_item[-2].replace("(", "")
+            date_item.append(f"{datetime.today().year}")
+
+        elif len(date_item) == 3:
+            date_item.append("09:00 am")
+            date_item.append(f"{datetime.today().year}")
+
+        return " ".join(date_item)
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date_item = self._clean_date_item(item[0])
+        date_obj = datetime.strptime(date_item, "%A, %B %d %H:%M %p %Y")
+
+        return date_obj
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        return {
+            "address": "Lincoln Park Chamber of Commerce, 2468 N. Lincoln, Chicago",
+            "name": "Lincoln Park Chamber of Commerce",
+        }
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        href = item[1].css("::attr(href)").get()
+        title = item[1].css("::text").get()
```

Comment:
The date isn't very relevant if we're already associating it with a meeting on a given date, so we should label it `Minutes` or `Agenda` based off of the section or URL text

Reply:
Could you elaborate a little more? I didn't understand @pjsier

Reply:
Sure, if we're looking at the document in the context of a meeting we already know the meeting date, so it isn't adding information as the document title. It's more helpful to see whether we're looking at an Agenda or Minutes, so ideally we could check which section of links we're in and assign the title that way, otherwise it looks like "Agenda" or "Minutes" are often included in the URL string so we could parse them that way

Reply:
Oh, i see. I'll check that out and make all the changes. Thank you for the explanation